src/resources/fulfillment_event.coffee

```diff
@@ -0,0 +1,34 @@
+BaseChild = require './base_child'
+pluralize = require 'pluralize'
+
+class FulfillmentEvent extends BaseChild
+  parent: '/orders'
+  slug:
```

Comment:
`slug` is an `object` here because according to the [docs](https://docs.shopify.com/api/fulfillmentevent#create) this resource has 2 different slugs: `event` for the request and `fulfillment_event` for the response.